### ISSUE TITLE
Change textColor method behavior on MessagesDisplayDelegate.

### ISSUE THREAD

**Issue by SD10** [MessageKit/MessageKit#624]
Currently, the `textColor(for:at:in)` message of `MessagesDisplayDelegate` applies to both the `.text(String)` and `.attributedText(NSAttributedString)` cases of `MessageData`.

This is a source of confusion when an`.attributedText` message's text color get's overridden by the delegate method. I think I originally set it up like this to avoid confusion but IMO this is not good behavior as it is still confusing and limits what can be done with `.attributedText`.

**TL;DR** we need to make `textColor(for:at:in)` only apply to `.text` messages.

**Comment by zhongwuzw** [comment]
Done in #625 .
